ranger-plugin/conf/ranger-servicedef-hawq.json

```diff
@@ -244,7 +244,7 @@
       "name": "authentication",
       "type": "enum",
       "subType": "authType",
-      "mandatory": false,
+      "mandatory": true,
```

Comment:
We have decided a while ago to not touch pg_hba.conf in Ambari, because users might have their own entries there and this is a very sensitive file. A viable approach might've been to manage all entries in pg_hba.conf via Ambari, but that required extra work.